test/utils/signed-request.ts

```diff
@@ -0,0 +1,104 @@
+import { AuthIdentity, IdentityType, Authenticator } from '@dcl/crypto'
+import { createUnsafeIdentity } from '@dcl/crypto/dist/crypto'
+
+export type SignedRequestOptions = {
+  method: string
+  path: string
+  body?: unknown
+  identity?: AuthIdentity
+  realAccount?: IdentityType
+  ephemeralIdentity?: IdentityType
+}
+
+export async function createSignedRequest(options: SignedRequestOptions): Promise<{
```

Comment:
Would you mind using the [decentraland-crypto-fetch](https://github.com/decentraland/decentraland-crypto-fetch) library to perform signed requests?